site/docs/red-team/configuration.md

```diff
@@ -0,0 +1,187 @@
+
+## Getting Started
+
+To initialize a basic redteam configuration, run:
+
+```bash
+npx promptfoo@latest redteam init
+```
+
+## Configuration Structure
+
+The following YAML structure outlines the configuration options available for red teaming:
+
+```yaml
+redteam:
+  plugins: Array<string | { name: string, numTests?: number }>
+  numTests: number # default number of tests to generate per plugin
+  injectVar: string | string[] # variables to inject
+  provider: string # test generation provider
+  purpose: string # purpose override string
+
+# Note that prompts and providers are still necessary for the generation of adversarial inputs.
+prompts:
+ - 'You are a helpful assistant that can answer questions. Answer in a friendly manner: {{question}}'
+providers:
+ - openai:gpt-3.5-turbo
+```
+
+### Fields
+
+| Field       | Type               | Description                                              | Default               |
+| ----------- | ------------------ | -------------------------------------------------------- | --------------------- |
+| `injectVar` | string \| string[] | Variable(s) to inject.                                   | Inferred from prompts |
+| `numTests`  | number             | Default number of tests to generate per plugin.          | 5                     |
+| `plugins`   | array              | Plugins to use for redteam generation.                   | Default plugins       |
+| `provider`  | string             | Provider used for generating adversarial inputs.         | Optional              |
+| `purpose`   | string             | Purpose override string describing the prompt templates. | Optional              |
+
+## Plugins
+
+Plugins are specified as an array of either strings (plugin names) or objects with `name` and optional `numTests` properties. They must exactly match the plugin names available in the redteam system.
+
+### Plugin Specification
+
+- **As a string**: `"plugin-name"`
+- **As an object**: `{ name: "plugin-name", numTests: 10 }`
+
+If `numTests` is not specified for a plugin, it will use the global `numTests` value.
+
+### Available Plugins
+
+Run the help menu to see a complete list of available plugins. Some common plugins include:
+
+- `competitors`
+- `harmful`
+- `jailbreak`
+- `overreliance`
+- `pii`
+- `prompt-injection`
+
+Note that plugins broadly fall into three categories:
+
+- individual plugins such as `overreliance`. This respect the `numTests` field.
+- categories such as `harmful`. These expand into multiple sub-categories, where each sub-category is a separate plugin.
+- methods such as `jailbreak` which are modifiers applied to other plugins.
+
+### Special Handling for 'harmful' Plugin
+
+When you include `harmful` in your plugins list, it will automatically include all available harm categories. You can also specify individual harm categories directly.
```

Comment:
is this now also the case for `pii` as well?

edit: seems you mention it in a separate section below.  Let's consolidate those sections